invokeai/app/services/config/config_default.py

```diff
@@ -280,6 +280,9 @@ class InvokeAIAppConfig(InvokeAISettings):
     deny_nodes          : Optional[List[str]] = Field(default=None, description="List of nodes to deny. Omit to deny none.", json_schema_extra=Categories.Nodes)
     node_cache_size     : int = Field(default=512, description="How many cached nodes to keep in memory", json_schema_extra=Categories.Nodes)
 
+    # MODEL IMPORT
+    civitai_api_key       : Optional[str] = Field(default=os.environ.get("CIVITAI_API_KEY"), description="API key for CivitAI", json_schema_extra=Categories.Other)
```

Comment:
Does everybody set this as an environment variable?

Reply:
The environment variable, set it in the config file, or pass it via command line flags.

Reply:
If it's unset, the config property is just None